modules/gapi/src/backends/cpu/gcpubackend.hpp

```diff
@@ -33,6 +33,7 @@ class GCPUExecutable final: public GIslandExecutable
 {
     const ade::Graph &m_g;
     GModel::ConstGraph m_gm;
+    cv::GCompileArgs m_compileArgs;
```

Comment:
I think it can be `const` as well.

Having `const` is a good way to let people reading this code know that this variable won't change once initialized - so will not cause any side effects going further.

Reply:
Thanks! Fixed.